usaspending_api/disaster/management/commands/generate_covid19_download.py

```diff
@@ -41,7 +43,42 @@ class Command(BaseCommand):
     readme_path = Path(settings.COVID19_DOWNLOAD_README_FILE_PATH)
     full_timestamp = datetime.strftime(datetime.now(timezone.utc), "%Y-%m-%d_H%HM%MS%S%f")
 
+    # KEY is the type of compute supported by this command
+    # key's VALUE are the strategies required by the compute type
+    # These strategies are used to change the behavior of this command
+    #   at runtime.
+    compute_types = {
+        ComputeTypeEnum.POSTGRES.value: {
+            "source_sql_strategy": {
+                "disaster_covid19_file_a": "usaspending_api/disaster/management/sql/disaster_covid19_file_a.sql",
+                "disaster_covid19_file_b": "usaspending_api/disaster/management/sql/disaster_covid19_file_b.sql",
+                "disaster_covid19_file_d1_awards": "usaspending_api/disaster/management/sql/disaster_covid19_file_d1_awards.sql",
+                "disaster_covid19_file_d2_awards": "usaspending_api/disaster/management/sql/disaster_covid19_file_d2_awards.sql",
+                "disaster_covid19_file_f_contracts": "usaspending_api/disaster/management/sql/disaster_covid19_file_f_contracts.sql",
+                "disaster_covid19_file_f_grants": "usaspending_api/disaster/management/sql/disaster_covid19_file_f_grants.sql",
+            },
+            "download_to_csv_strategy": PostgresToCSVStrategy(logger=logger),
+        },
+        ComputeTypeEnum.DATABRICKS.value: {
```

Comment:
See previous comment about avoiding use of "Databricks" in names.

Reply:
Addressed.